api/v1/models/Attendee.js

```diff
@@ -45,20 +45,19 @@ const Attendee = Model.extend({
     isNovice: ['required', 'boolean'],
     isPrivate: ['required', 'boolean'],
     phoneNumber: ['string', 'maxLength:15'],
-    acceptanceType: ['string', validators.in(ACCEPTANCE_TYPES)],
-    acceptedEcosystemId: [ 'integer' ]
+    acceptanceType: ['string', validators.in(ACCEPTANCE_TYPES)]
   },
   interests: function() {
     return this.hasMany(AttendeeProjectInterest);
   },
-  projects: function() {
-    return this.hasMany(AttendeeProject);
+  longform: function() {
+    return this.hasMany(AttendeeExtraInfo);
```

Comment:
See above re: model usage

Reply:
Fixed in cd52938e2c67991a451212c0cd3d2fa14e1edbe7